collections/VideosCollection.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable new-cap */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+const VideosCollection = Backbone.Collection.extend({
+  model: VideoModel,
+
+  parse(response) {
+    _.each(response.items, (video) => {
+      const videoModel = new this.model();
+
```

Comment:
I'd probably prefer a simple `.map` here - be careful about an over-reliance on Lodash.